src/main/scala/se/nimsa/dcm4che/streams/DicomParsing.scala

```diff
@@ -19,13 +19,30 @@ package se.nimsa.dcm4che.streams
 import akka.util.ByteString
 import org.dcm4che3.data.{ElementDictionary, VR}
 import org.dcm4che3.io.DicomStreamException
+import org.dcm4che3.data.UID._
+import org.dcm4che3.data.Tag._
 
 /**
   * Helper methods for parsing binary DICOM data.
   */
 trait DicomParsing {
 
-  case class Info(bigEndian: Boolean, explicitVR: Boolean, hasFmi: Boolean)
+  case class Info(bigEndian: Boolean, explicitVR: Boolean, hasFmi: Boolean) {
+    /**
+      * Best guess for transfer syntax.
+      * @return transfer syntax uid value
+      */
+    def assumedTransferSyntax = if (explicitVR) {
+      if (bigEndian) {
+        ExplicitVRBigEndianRetired
+      } else {
+        ExplicitVRLittleEndian
+      }
+    } else {
+      ImplicitVRLittleEndian
+    }
+  }
+  case class DataElement(tag: Int, vr: VR, length: Int, value: ByteString)
```

Comment:
Changed from 'Attribute' to 'DataElement' after reading this:
ftp://dicom.nema.org/medical/DICOM/2013/output/chtml/part05/sect_3.10.html
Maybe 'Attribute' is better anyway?